presto-hive/src/test/java/com/facebook/presto/hive/AbstractTestHiveClient.java

```diff
@@ -2942,7 +2942,8 @@ private void doInsertIntoExistingPartitionEmptyStatistics(HiveStorageFormat stor
 
             for (String partitionName : partitionNames) {
                 HiveBasicStatistics statistics = getBasicStatisticsForPartition(transaction, tableName, dsColumn, partitionName);
-                assertStatisticsAreNotPresent(statistics);
+                assertThat(statistics.getRowCount()).isNotPresent();
+                assertThat(statistics.getInMemoryDataSizeInBytes()).isNotPresent();
```

Comment:
Add comment that `fileCount` and `onDiskDataSizeInBytes` is deliberately not checked here.

In addition, can you explain in more details why this is the case. (I'm asking for my understanding of this diff. But it would also be helpful if you put that in the code comment.)